README.md

```diff
@@ -65,6 +65,7 @@ table includes the currently implemented options.
 | Option  | Sample Value  | Description|
 |---|---|---|
 | async  | N/A  | Run the project without waiting for results. The TestJobID will be printed on the console to make it possible to query status, get reports or cancel the job|
+| printReport  | N/A  | Print the execution report to the console for synchronous job after it is completed |
```

Comment:
I think you should explain both scenarios. Both synchronous and printing it after it has run.

Reply:
This one is  when the job is run synchronously. And to print the report for already run job is different which I have written already.

Reply:
ah, ok!